pkg/trisa/api/v1beta1/mock/directory.go

```diff
@@ -0,0 +1,126 @@
+package mock
+
```

Comment:
Not sure if this is the appropriate place for the mocks but it seemed to make sense to me.

Reply:
This works for now - thank you!